README.rst

```diff
@@ -146,8 +146,8 @@ The general synopsis is as follows:
 
    Usage: cardano-cli (Era based commands | Byron specific commands | Miscellaneous commands)
 
-> NOTE: the exact invocation command depends on the environment.  If you have only built ``cardano-cli``, without installing it, then you have to prepend :code:`cabal run -- `
-before :code:`cardano-cli`.  We henceforth assume that the necessary environment-specific adjustment has been made, so we only mention ``cardano-cli``.
+> NOTE: the exact invocation command depends on the environment.  If you have only built ``cardano-cli``, without installing it, then you have to prepend ``cabal run -- ``
```

Comment:
Hi, @uwe, it seems like even after this change, the markdown parser is still somewhat confused.

I think the output you want is: If you have only built `cardano-cli` without installing it, then you have to prepend `cabal run -- ` before `cardano-cli`.

Which you can achieve with this:

```markdown
If you have only built `cardano-cli` without installing it, then you have to prepend `cabal run --` before `cardano-cli`.
```

For clarity and readability, though, you might consider simplifying the sentence by saying:

```markdown
If you've built `cardano-cli` without installing it, you should instead use `cabal run -- cardano-cli`.
```

Reply:
@newhoggy Preskton is right, this is still broken. Shall I make a PR with his proposal (last sentence)? Or do you want to fix this directly on master?

Reply:
I'm happy to raise a PR. 👍🏼

Reply:
Raised #2522.